fabric-command-api-v1/src/main/java/net/fabricmc/fabric/mixin/command/client/ClientCommandSourceMixin.java

```diff
@@ -38,6 +44,12 @@
 	@Final
 	private MinecraftClient client;
 
+	private Entity entity = client.player;
+	private Vec3d position = entity.getPos();
+	private Vec2f rotation = entity.getRotationClient();
+	private ClientWorld world = client.world;
+	private Map<String, Object> meta = new HashMap<>();
```

Comment:
These fields should be removed. The non-stub implementations would happen in mod-defined custom client command sources.